apps/core/management/commands/load_catalog.py

```diff
@@ -72,6 +73,18 @@ def handle(self, *args, **options):
                                     destination,
                                 )
 
+                        elif settings.EVILFLOWERS_STORAGE_DRIVER == "apps.files.storage.s3.S3Storage":
+                            tar.extractall(path=temp_dir)
+                            storage = get_storage()
+                            extracted_storage_path = os.path.join(temp_dir, "storage")
+                            for catalog_name in catalog_names:
+                                source = os.path.join(extracted_storage_path, f"catalogs/{catalog_name}")
+                                destination = os.path.join(settings.EVILFLOWERS_STORAGE_S3_BUCKET, f"catalogs/{catalog_name}")
+                                for root, subdirs, files in os.walk(source):
+                                    entry_id = root.split(f"{catalog_name}/")[-1]
+                                    for file in files:
+                                        storage._save_from_path(os.path.join(destination, entry_id, file), os.path.join(root, file))
```

Comment:
You are not suppose to access private method from outside. Also, I am not a big fan of accessing storage directly